### DISCUSSION TITLE
used winutil to create a debloated .iso - ended up with no browser

### DISCUSSION THREAD

**Discussion by whatisyourname69** [ChrisTitusTech/winutil#1902] [Q&A]
Hey Guys, 

I recognize this is probably a user error rather than an issue - I used the toolbox to create the debloated W11 iso but I ended up with no browser or the ability to install any apps via the toolbox itself - I think it was missing some dependencies to be able to download stuff using the toolbox. 

How do I make sure that there is some form of a browser installed - perhaps the toolbox should force a install of a browser of choice. Just an idea? 

Any thoughts welcome.

**Comment by UltraCenterHQ** [comment]
use a usb flash drive or connect your phone to the pc to transfer a browser setup file

**Reply by whatisyourname69** [reply]
agreed that is a possibility, however, would it not be possible to opt to have "a browser of choice" added to the .iso file created by the custom .iso script. 

An optional rather than a forced feature
